score/mw/com/runtime_configuration.cpp

```diff
@@ -79,6 +80,21 @@ std::optional<filesystem::Path> RuntimeConfiguration::ParseConfigurationPath(
             }
             return score::cpp::at(command_line_args, static_cast<std::ptrdiff_t>(index_of_configuration_path));
         }
+        if (command_line_argument_key == kDeprecatedConfigurationPathCommandLineKey)
+        {
+            score::mw::log::LogWarn("lola")
+                    << "Command line argument" << kDeprecatedConfigurationPathCommandLineKey 
+                    << "is deprecated, please use" << kConfigurationPathCommandLineKey;
+            const auto index_of_configuration_path = arg_idx + 1U;
+            if (index_of_configuration_path >= num_args)
+            {
+                score::mw::log::LogFatal("lola")
```

Comment:
I think the positive branch of the `if-statement` is not covered.

I think we need an additional test similar to: `TerminatesIfCommandLineArgsContainPathKeyButNoPath`

Reply:
Added additional test case to cover that branch and additional asserts for checking the termination reason is in log output.